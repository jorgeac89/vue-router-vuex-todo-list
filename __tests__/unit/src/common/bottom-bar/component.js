import { shallowMount } from '@vue/test-utils';
import BottomBar from 'common/bottom-bar';

describe('BottomBar component', () => {
  it('renders correctly', () => {
    expect(shallowMount(BottomBar)).toMatchSnapshot()
  });
});
