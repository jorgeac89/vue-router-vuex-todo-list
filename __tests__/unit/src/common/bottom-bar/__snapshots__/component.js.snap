// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomBar component renders correctly 1`] = `
<div class="container-fluid bg-dark text-light mt-5">
  <div class="row py-2">
    <div class="col-12 d-flex flex-column flex-md-row justify-content-center text-center text-nowrap">
      <div><span class="mx-3">Jorge Ant√≥n Caballero</span> <a href="mailto:jorgeac89@gmail.com" class="text-reset text-decoration-none mx-3"><i class="fas fa-envelope"></i>
          &nbsp;
          jorgeac89@gmail.com
        </a></div>
      <div class="mt-2 mt-md-0"><a href="https://github.com/jorgeac89/" class="text-reset text-decoration-none mx-3"><i class="fab fa-github"></i>
          &nbsp;
          Github
        </a> <a href="https://www.linkedin.com/in/jorge-anton-caballero" class="text-reset text-decoration-none mx-3"><i class="fab fa-linkedin-in"></i>
          &nbsp;
          Linkedin
        </a></div>
    </div>
  </div>
</div>
`;
